tests/test_all_plugins.py

```diff
@@ -0,0 +1,33 @@
+import os
+from typing import List
+
+import pytest
+
+PLUGIN = os.getenv("TEST_PACKAGE_NAME")
+if not PLUGIN:
+    pytest.skip("skipping plugin specific tests", allow_module_level=True)
+
+try:
+    from importlib import metadata
+except ImportError:
+    import importlib_metadata as metadata  # type: ignore
+
+
+@pytest.fixture
+def entry_points():
+    d = metadata.distribution(str(PLUGIN))
+    return [
+        ep for ep in d.entry_points if ep.group in ("napari.plugin", "napari.manifest")
+    ]
+
+
+def test_plugin_has_entry_points(entry_points):
+    if PLUGIN not in {"napari-console", "napari-error-reporter"}:
```

Comment:
why not error reporter?

Reply:
Because like console, it doesn't have any hook implementations.  It's a "blessed" plugin with direct injection logic in napari